<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableItemDialog</class>
 <widget class="QDialog" name="tableItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>270</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>460</width>
    <height>210</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label1">
      <property name="text">
       <string>列  号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBoxColumn">
      <property name="minimum">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label2">
      <property name="text">
       <string>行  号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBoxRow"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label3">
      <property name="text">
       <string>设定文字</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="edtCaption"/>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="btnSetText">
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设定文字</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/506.bmp</normaloff>:/images/506.bmp</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="text">
     <string>关  闭</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/132.bmp</normaloff>:/images/132.bmp</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>420</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
